homeassistant/components/device_tracker/ddwrt.py

```diff
@@ -84,12 +86,12 @@ def get_device_name(self, device):
                     return None
 
                 # Remove leading and trailing single quotes.
-                cleaned_str = dhcp_leases.strip().strip('"')
-                elements = cleaned_str.split('","')
-                num_clients = int(len(elements)/5)
+                cleaned_str = dhcp_leases.replace("\'", "").replace(" ", "")
```

Comment:
I do not have double quotes in [my data](https://github.com/home-assistant/home-assistant/pull/3996/files#diff-7e86b2b8efdcb26c4ebf3929a26b0686R12). For the unlikely case that a different firmware version has double quotes I will just add this. To be on the safe side.